waspc/cli/src/Wasp/Cli/Command/CreateNewProject/StarterTemplates.hs

```diff
@@ -80,7 +81,7 @@ defaultStarterTemplate =
       }
 
 findTemplateByString :: [StarterTemplate] -> String -> Maybe StarterTemplate
-findTemplateByString templates query = find ((== query) . show) templates
+findTemplateByString templates query = find ((query `isPrefixOf`) . show) templates
```

Comment:
@infomiho I made the change both here, and in Interactive, so that matches happen on prefix, not on the exact match, because I thought that will be more practical for users + it allowed to putting stuff like emojis at the end of the template title (one for AI).

Reply:
What if we have some sort of overlap with the names like `saas-minimal` and `saas-stripe`?

Reply:
Then it will take the first one, which I think is ok?